ledger/ledger-api-test-tool/BUILD.bazel

```diff
@@ -264,6 +264,37 @@ conformance_test(
     ],
 )
 
+conformance_test(
+    name = "conformance-test-tls1.2-or-newer",
+    extra_data = [
+        "//ledger/test-common/test-certificates:client.crt",
+        "//ledger/test-common/test-certificates:client.pem",
+        "//ledger/test-common/test-certificates:server.crt",
+        "//ledger/test-common/test-certificates:server.pem",
+        "//ledger/test-common/test-certificates:server.pem.enc",
+        "//ledger/test-common/test-certificates:ca.crt",
+    ],
+    lf_versions = lf_version_configuration_versions,
+    ports = [6865],
+    server = "//ledger/ledger-on-memory:app",
+    server_args = [
+        "--contract-id-seeding=testing-weak",
+        "--participant=participant-id=example,port=6865",
+        "--crt $$(rlocation $$TEST_WORKSPACE/$(rootpath //ledger/test-common/test-certificates:server.crt))",
+        "--cacrt $$(rlocation $$TEST_WORKSPACE/$(rootpath //ledger/test-common/test-certificates:ca.crt))",
+        "--pem $$(rlocation $$TEST_WORKSPACE/$(rootpath //ledger/test-common/test-certificates:server.pem.enc))",
+        "--tls-secrets-url https://raw.githubusercontent.com/digital-asset/daml/main/ledger/test-common/files/server-pem-decryption-parameters.json",
+        "--min-tls-version 1.2",
```

Comment:
Once Netty upgrade PR is merged this test shall pass without `--min-tls-version 1.2`.

Alternatively we can code defensively and filter out protocols lower than TLS 1.1 before they are passed to initialize server's SSL context. And/or add an assertion to fail fast when server's SSL context has been started with TLS 1.1 or lower enabled.
I haven't added such assertion because I saw value in starting a server with e.g. TLS 1.1 enabled and seeing that can connect with a client on a TLS 1.1. And subsequently seeing that a server on TLS 1.2 fails with the same client.